src/specfem3D/write_seismograms.f90

```diff
@@ -58,38 +58,30 @@ subroutine write_seismograms()
     if (GPU_MODE) then
       ! gets field values from GPU
       ! this transfers fields only in elements with stations for efficiency
-      if (SIMULATION_TYPE == 2) then
```

Comment:
not sure if this change is intended. it affects the wavefield transfer now also for typical forward and kernel simulations.